backend/portal/api/portal-api.yml

```diff
@@ -702,7 +750,88 @@ paths:
                       type: number
         "400":
           $ref: "#/components/responses/400"
-
+  /v1/user-datasets/index:
+    get:
+      tags:
+        - datasets
+      summary: >-
+        Returns all public and private, non tombstoned datasets where the user has WRITE access. This endpoint is meant to be used for displaying datasets
+        in a compact view. It doesn't return all the fields, so it is not suitable for general usage.
```

Comment:
same thing; update summary

Reply:
Although it appears actually you have added the `WRITE access` detail already... is the summary otherwise as you intend it?

Reply:
@NoopDog confirm

Reply:
This is updated thanks.